spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/context/MessageSourceProperties.java

```diff
@@ -42,6 +42,11 @@ public class MessageSourceProperties {
 	 */
 	private String basename = "messages";
 
+	/**
+	 * Comma-separated list of locale-independent common messages.
+	 */
+	private String commonMessages;
```

Comment:
I think we might be better off using a `List<Resource>` here, that way the `Binder` can take care of creating the resource instances.